samples/ImplicitFlow01/AppWithFetchClient/wwwroot/app.js

```diff
@@ -0,0 +1,72 @@
+/* global window, fetch, Oidc */
+
+//
+// Define helper functions.
+//
+const fetchUserResources = (accessToken) => {
+  const url = `${window.location.origin}/api`;
+  const options = {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  };
+
+  // See https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+  fetch(url, options)
+    .then(response => response.json())
+    .then(obj => JSON.stringify(obj))
+    .then((json) => {
+      window.document.getElementById('fetch_response').innerText = json;
+    })
+    .catch(err => console.log(err)); // eslint-disable-line no-console
```

Comment:
Since your displaying stringified JSON to the UI, this error could also be a stringified JSON to the UI

Reply:
Good idea. Done. I also factored out the `window.document...` into a `writeInnerText` function.